### PR TITLE
[DO NOT MERGE] feat: add EntryList files to examples [BAU-648]

### PR DESCRIPTION
**Pull request by massao**
Add EntryList example

### CODE REVIEW

File: examples/entry-list/src/index.js
Comment (bgutsol) [inline examples/entry-list/src/index.js:10]: I guess there is no need to return `null` in case it's not `LOCATION_ENTRY_LIST`
```suggestion
init((sdk) => {
  if (sdk.location.is(locations.LOCATION_ENTRY_LIST)) {
    sdk.entryList.onEntryListUpdated((props) => onEntryListUpdated({entries: props.entries}))
  }
});
```

File: examples/entry-list/src/locations/EntryList.js
Comment (bgutsol) [inline examples/entry-list/src/locations/EntryList.js:1]: maybe let's make it `async` just to show that the listener can accept the promise as well, wdyt?

**Review by bgutsol — COMMENTED** [review]
LGTM, only a couple of comments, no blockers.

Won't approve, because I'm not sure if we can merge it before public release.

@andipaetzold should we wait for the public release before merging it?

**Comment by massao** [comment]
> LGTM, only a couple of comments, no blockers.
> 
> Won't approve, because I'm not sure if we can merge it before public release.
> 
> @andipaetzold should we wait for the public release before merging it?

Good point, I think we need to wait for the public release, because I'm even using the alpha version, and in some casees had to manually update the `node_modules` because of the `@contentful/react-apps-toolkit`

File: examples/entry-list/README.md
Comment (andipaetzold) [inline examples/entry-list/README.md:16]: Change the port to 3000

File: examples/entry-list/package.json
Comment (andipaetzold) [inline examples/entry-list/package.json:7]: Parcel is a dev dependency

File: examples/javascript/src/locations/EntryList.jsx
Comment (andipaetzold) [inline examples/javascript/src/locations/EntryList.jsx:23]:
```suggestion
  }, [])
```

File: examples/javascript/src/locations/EntryList.jsx
Comment (andipaetzold) [inline examples/javascript/src/locations/EntryList.jsx:22]: Does `onEntryListUpdated` return an unsubscribe method? If yes, we should add a `return` here

**Review by andipaetzold — CHANGES_REQUESTED** [review]

File: examples/javascript/src/locations/EntryList.jsx
Comment (bgutsol) [reply examples/javascript/src/locations/EntryList.jsx:22]: It doesn't return an unsubscribe method

File: examples/entry-list/README.md
Comment (andipaetzold) [inline examples/entry-list/README.md:16]:
```suggestion
To test, you can create an app definition in your Contentful organization settings pointing to `http://localhost:3000` and registering the `entry-list` location.
```

**Review by andipaetzold — CHANGES_REQUESTED** [review]
Please use lockfileversion 1 to keep consistent with the rest of the repo

Otherwise, this looks good.

File: examples/javascript/src/locations/EntryList.jsx
Comment (andipaetzold) [inline examples/javascript/src/locations/EntryList.jsx:5]: should be
```
  const sdk = useSDK<EntryListExtensionSDK>();
```

File: examples/typescript/src/locations/EntryList.tsx
Comment (andipaetzold) [inline examples/typescript/src/locations/EntryList.tsx:23]:
```suggestion
    sdk.entryList.onEntryListUpdated(onEntryListUpdated)
```

File: examples/javascript/src/locations/EntryList.jsx
Comment (andipaetzold) [inline examples/javascript/src/locations/EntryList.jsx:22]:
```suggestion
    sdk.entryList.onEntryListUpdated(onEntryListUpdated)
```

**Review by andipaetzold — APPROVED** [review]

File: examples/references-count/src/locations/EntryList.ts
Comment (andipaetzold) [inline examples/references-count/src/locations/EntryList.ts:9]: Can we just use `sdk.locales.default` instead?

Comment (bgutsol) [reply examples/references-count/src/locations/EntryList.ts:9]: YES! I didn't know we have it

**Pull request closed without merging**
